<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataPlotlyDialogBase</class>
 <widget class="QDialog" name="DataPlotlyDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Plotly</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Plot Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="plot_combo"/>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTabWidget" name="tab_property">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="prop_tab">
      <attribute name="title">
       <string>Plot Properties</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>324</width>
            <height>349</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QgsCollapsibleGroupBox" name="mGroupBox_2">
             <property name="title">
              <string>Properties</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="3" column="0">
               <widget class="QLabel" name="alpha_lab">
                <property name="text">
                 <string>Transparency</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="2">
               <widget class="QSlider" name="alpha_slid">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksAbove</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="in_color_lab">
                <property name="text">
                 <string>Marker Color</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="out_color_lab">
                <property name="text">
                 <string>Ouline Color</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="3">
               <widget class="QComboBox" name="marker_type_combo"/>
              </item>
              <item row="3" column="3">
               <widget class="QSpinBox" name="alpha_num">
                <property name="maximum">
                 <number>100</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QgsColorButton" name="out_color_combo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="marker_size_lab">
                <property name="text">
                 <string>Marker Size</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="marker_type_lab">
                <property name="text">
                 <string>Marker type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="marker_width_lab">
                <property name="text">
                 <string>Marker Width</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QSpinBox" name="marker_width">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QgsColorButton" name="in_color_combo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSpinBox" name="marker_size">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
             <zorder>out_color_lab</zorder>
             <zorder>marker_width</zorder>
             <zorder>marker_type_lab</zorder>
             <zorder>marker_width_lab</zorder>
             <zorder>alpha_num</zorder>
             <zorder>alpha_slid</zorder>
             <zorder>in_color_lab</zorder>
             <zorder>out_color_combo</zorder>
             <zorder>marker_size</zorder>
             <zorder>in_color_combo</zorder>
             <zorder>marker_size_lab</zorder>
             <zorder>alpha_lab</zorder>
             <zorder>marker_type_combo</zorder>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
             <property name="title">
              <string>Plot Parameters</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="layer_label">
                <property name="text">
                 <string>Layer</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QgsMapLayerComboBox" name="layer_combo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="x_label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>X Field</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QgsFieldExpressionWidget" name="x_combo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="y_label">
                <property name="text">
                 <string>Y Field</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QgsFieldExpressionWidget" name="y_combo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
          <zorder>mGroupBox</zorder>
          <zorder>mGroupBox_2</zorder>
          <zorder>verticalSpacer</zorder>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="custom_tab">
      <attribute name="title">
       <string>Plot Customizations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>310</width>
            <height>373</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="3" column="1" colspan="3">
            <widget class="QLineEdit" name="x_axis_title"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="bar_mode_lab">
             <property name="text">
              <string>Bar Mode</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QLineEdit" name="plot_title_line"/>
           </item>
           <item row="7" column="1" colspan="3">
            <widget class="QComboBox" name="bar_mode_combo"/>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="show_legend_check">
             <property name="text">
              <string>Show Legend</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="1" rowspan="2" colspan="3">
            <widget class="QLineEdit" name="y_axis_title"/>
           </item>
           <item row="11" column="0" colspan="4">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>255</width>
               <height>222</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="orientation_label">
             <property name="text">
              <string>Bar orientation</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="3">
            <widget class="QComboBox" name="orientation_combo"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="box_statistic_label">
             <property name="text">
              <string>Show Statistics</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1" colspan="3">
            <widget class="QComboBox" name="box_statistic_combo"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="plot_title_lab">
             <property name="text">
              <string>Plot Title</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="x_axis_label">
             <property name="text">
              <string>X Label</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" rowspan="2">
            <widget class="QLabel" name="y_axis_label">
             <property name="text">
              <string>Y Label</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="outliers_label">
             <property name="text">
              <string>Outliers</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1" colspan="3">
            <widget class="QComboBox" name="outliers_combo"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="bar_legend_label">
             <property name="text">
              <string>Field Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QLineEdit" name="bar_legend_title"/>
           </item>
           <item row="0" column="3">
            <widget class="QCheckBox" name="range_slider_combo">
             <property name="text">
              <string>Show RangeSlider</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="hist_norm_label">
             <property name="text">
              <string>Normalization</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="3">
            <widget class="QComboBox" name="hist_norm_combo"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="plot_tab">
      <attribute name="title">
       <string>Plot Table</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>188</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QTableWidget" name="traceTable">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <column>
          <property name="text">
           <string>Trace ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Plot Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="remove_button">
         <property name="text">
          <string>Remove Trace From List</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Help</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type of Plot</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="subcombo"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QRadioButton" name="radio_rows">
     <property name="text">
      <string>Plot in Rows</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QRadioButton" name="radio_columns">
     <property name="text">
      <string>Plot in Columns</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="draw_btn">
     <property name="text">
      <string>Add Plot</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="addTrace_btn">
     <property name="text">
      <string>Draw Plot</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="clear_btn">
     <property name="text">
      <string>Clear Plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgis.gui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QPushButton</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>alpha_slid</sender>
   <signal>valueChanged(int)</signal>
   <receiver>alpha_num</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alpha_num</sender>
   <signal>valueChanged(int)</signal>
   <receiver>alpha_slid</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layer_combo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>x_combo</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layer_combo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>y_combo</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x_combo</sender>
   <signal>fieldChanged(QString)</signal>
   <receiver>x_axis_title</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y_combo</sender>
   <signal>fieldChanged(QString)</signal>
   <receiver>y_axis_title</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y_combo</sender>
   <signal>fieldChanged(QString)</signal>
   <receiver>bar_legend_title</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
